.github/dependabot.yaml

```diff
@@ -353,3 +353,136 @@ updates:
       actions:
         patterns:
           - "actions/*"
+
+  # Below configuration is workaround for dependabot issue
+  # about local actions in .github/actions/ are not checked
+  # https://github.com/dependabot/dependabot-core/issues/6345
+
+  - package-ecosystem: github-actions
+    directory: /.github/actions/cache-go-dependencies
+    schedule:
+      interval: 'weekly'
+      day: 'wednesday'
+    open-pull-requests-limit: 1
```

Comment:
I find the combination of weekly and single PR allowed could lead to some important updates being missed or getting queued. IMHO, a value of 3 (applicable to all added trees) would not be too problematic.

Can be handled as follow-up action and be extended to other directories (except maybe with the UI, to be checked with the impacted team).

Reply:
Security is applied without a queue but I can change that.

Reply:
Fixed